src/inverseRobot.js

```diff
@@ -8,6 +8,19 @@
 
 function inverseRobot(robot) {
   // write code here
+  const inverted = {};
+  const roKeys = Object.keys(robot);
+  const roVals = Object.values(robot);
```

Comment:
be more descriptive, don't be afraid to use full name, `ro` sounds mysterious

Reply:
It was originally this way to avoid the linter being mad about lines being long... although in hindsight it was unnecessary